### ISSUE TITLE
Client connected but it doesn't receive anything

### ISSUE THREAD

**Issue by pagoru** [socketio/socket.io-deno#4]
The client is connected fine and can send and receive events, but some times the server doesn't recieve any event and the client is still connected. 

The way to reproduce it:
Start and stop the client a few times, until it stops receiveing the `pong` message.

client: 
```ts
const socket = io("http://localhost:8080");

socket.on("pong", ({ datetime }) => {
    console.log(`Response time (${(Date.now() - datetime)}ms)`)
});

socket.on("connect", () => {
    console.log(`Connected! ${socket.id}`);
    
    setInterval(() => {
        socket.emit('ping', { datetime: Date.now() });
    }, 1000);
});

socket.connect();
```

server: 
```ts
const io = new Server();

io.on("connection", (socket) => {
    console.log(`socket ${socket.id} connected`);
    
    socket.on('ping', (data) => {
        console.log(`[${socket.id}]: ping!`)
        socket.emit('pong', data)
    });
});

await serve(io.handler(), {
    port: 8080,
});
```

**Comment by pagoru** [comment]
Woooow, only two weeks from the start?? Great job, dude!

**Comment by darrachequesne** [comment]
This should be fixed by https://github.com/socketio/socket.io-deno/commit/c706741544e33ca364ef88e3779aa8d4ee3739f0, included in [0.2.0](https://github.com/socketio/socket.io-deno/releases/tag/0.2.0).

Good catch, thanks!

**Comment by sonickseven** [comment]
@pagoru can you share your full code from client to connect, I'm trying to connect, but never I have fabled connect :(
